src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/CreateTask.groovy

```diff
@@ -16,10 +16,51 @@
 package net.wasdev.wlp.gradle.plugins.tasks
 
 import net.wasdev.wlp.gradle.plugins.Liberty
+
+import java.io.File;
+
+import org.gradle.api.tasks.InputDirectory;
+import org.gradle.api.tasks.InputFile;
+import org.gradle.api.tasks.Optional;
+import org.gradle.api.tasks.OutputDirectory;
+import org.gradle.api.tasks.OutputFile;
 import org.gradle.api.tasks.TaskAction
 
+import groovy.lang.Closure;
+
 class CreateTask extends AbstractServerTask {
 
+	@InputDirectory @Optional
+	File getConfigDir() {
+		if(project.liberty.server.configDirectory != null && project.liberty.server.configDirectory)
```

Comment:
`project.liberty.server` can be changed to `server` for each of the instances in this file.